mongodb/netlify-extensions
# DOP-5022: Adding netlify integration for mut-publish

### Stories/Links

[DOP-5022](https://jira.mongodb.org/browse/DOP-5022)
[git-changed-files-testing](https://app.netlify.com/sites/git-changed-files-testing-bianca/deploys)
[bianca-fork-docs-landing](https://app.netlify.com/sites/bianca-fork-docs-landing/deploys)

### Notes / Blockers

* Everything currently works for the test case of enabling this integration on 'bianca-fork-docs-landing" in the s3 buck of **docs-mongodb-org-dotcomstg** under the prefix **/netlify/docs-qa**
    * While Brandon and I were pair programming we discussed that it was safe to use to the docs-mongodb-org-dotcomstg s3 bucket with that prefix since its the staging prod under a new prefix so it wouldn't mess up anything in prod
* I've added some comments throughout the code about stuff that maybe needs changing that I'm unsure of. Such as how many logs/comments I should be making throughout.
* In my `mut-publish` command I'm unsure of which bucket we would want to use in prod/what prefix should be used. I added more context in the comments of the code for this question and what should maybe be changed.
*  I know this is eventually going to be its own integration but right now I have added the integration to git-changed-files integration. However, I made a copy so the integration should be under **git-changed-files-testing** in netlify

### Testing
* Enable the git-changed-files-testing integration on a forked content repo
* deploy the content repo
* you should see the content in the linked s3 bucket
